AWSAuthSDK/Sources/AWSAuthUI/AWSAuthUIViewController.m

```diff
@@ -124,7 +125,16 @@ + (void)presentViewControllerWithNavigationController:(UINavigationController *)
 + (AWSAuthUIConfiguration *)getDefaultAuthUIConfiguration {
     
     AWSAuthUIConfiguration *authUIConfig = [[AWSAuthUIConfiguration alloc] init];
-    
+
+    if ([AWSAuthUIViewController isConfigurationKeyPresent:AWSInfoAppleIdentifier]) {
+        Class appleClass = NSClassFromString(@"AWSAppleSignInButton");
```

Comment:
When would this class be unavailable? If it's only platform version, then put appropriate availability wrappers around the API, and just directly call `[authUIConfig addAWSSignInButtonViewClass:[AWSAppleSignInButton class]]`